src/database/operations/assert/assertSourceExists.ts

```diff
@@ -1,5 +1,5 @@
 import { db } from '../../db';
-import { InputConflictError } from '../../../errors';
+import { NotFoundError } from '../../../errors';
```

Comment:
I guess we can't replace `InputConflictError` everywhere with `NotFoundError` because in some cases we simply surface the `NotFoundError` to the caller but in other cases it's a more complex situation of needing `entityXId` in order to post to `entityYId`, and so that's the `InputConflictError`?

Reply:
I figured it's just that operations are not written in any context (e.g. this assertion may be made on something that has nothing to do with input).

On the other side: our errors map to status codes (not found errors would be a 404), whereas input conflict error would map to a different 400, so we do need to map it to inputconflicterror